### ISSUE TITLE
Can I make redirect_uri unrequired on application creation?

### ISSUE THREAD

**Issue by lucasbesen** [danschultzer/ex_oauth2_provider#70]
Is there a way to make redirect_uri unrequired when creating/editing a new application?

Server to server auth doesn't need this

**Comment by danschultzer** [comment]
Yeah, it's [required in the ecto schema](https://github.com/danschultzer/ex_oauth2_provider/blob/v0.5.5/lib/ex_oauth2_provider/applications/application.ex#L29), but it does make sense to make it optional. Let me look at this when I get some free time.

The default would be to just set it as `urn:ietf:wg:oauth:2.0:oob`.
